bika/lims/content/worksheet.py

```diff
@@ -142,11 +142,6 @@ def _renameAfterCreation(self, check_auto_id=False):
     def Title(self):
         return safe_unicode(self.getId()).encode('utf-8')
 
-    def getFolderContents(self, contentFilter):
```

Comment:
Instead of deleting the function, consider to use `depracated` decorator. See https://github.com/naralabs/bika.lims/pull/32/files#r106821886 
We never know beforehand if someone is using it in their own customized add-on